scripts/image-builder/mirror-images.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
```

Comment:
If you mean `oc-mirror`, I reviewed [this docs](https://docs.openshift.com/container-platform/4.13/installing/disconnected_install/installing-mirroring-disconnected.html), but they do not seem to work because we're not supporting imagesets in MicroShift and we're unable to `init` the process.

Reply:
Can't this be done with podman pull and push?

Reply:
Maybe, but I'm not sure that there's a one command invocation for podman to do this. 
So, we need a script to make the procedure convenient. The usage of skopeo vs podman can be fixed, of course, if you think podman is "better"